data/pathfinder/paizo/roleplaying_game/advanced_class_guide/acg_abilities_class.lst

```diff
@@ -2810,7 +2846,7 @@ Rapier Weapon Mastery		KEY:Inspired Blade ~ Rapier Weapon Mastery		CATEGORY:Spec
 
 # Mouser
-Deeds					KEY:Mouser ~ Deeds						CATEGORY:Special Ability	TYPE:Mouser Class Feature.Swashbuckler Class Feature.SpecialQuality																																																																																																																																																																				DESC:The mouser gains the following deeds, each of which replaces an existing deed.																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																		SOURCEPAGE:p.126
+Deeds					KEY:Mouser ~ Deeds						CATEGORY:Special Ability	TYPE:Mouser Class Feature.Swashbuckler Class Feature.SpecialQuality																																																																																																																																																																				DESC:The mouser gains the following deeds, each of which replaces an existing deed.																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																											2|PREVARGTEQ:MagusLVL,2																																																																																																																																																																SOURCEPAGE:p.126
```

Comment:
It appears to be an artifact from a previous copy and paste. It can simply be deleted. I searched for it previously, but I must have missed it.